src/client/e2e/credit.spec.ts

```diff
@@ -4,37 +4,36 @@ import { test } from "./fixtures"
 import { OPENFIN_PROJECT_NAME } from "./utils"
 
 test.describe("Credit", () => {
-  test.describe("New RFQ", () => {
-    test("When I select Googl instrument and click Send RFQ button then I should see a GOOGL RFQ created on the RFQ sections and I can accept any value @smoke", async ({
-      context,
-      creditPagesRec,
-    }, testInfo) => {
-      test.setTimeout(120000)
+  let newRfqPage: Page
+  let rfqsPage: Page
+  let rfqBlotterPage: Page
 
-      let newRfqPage: Page
-      let rfqsPage: Page
-      let rfqBlotterPage: Page
+  test.beforeAll(async ({ context, creditPagesRec }, testInfo) => {
+    if (testInfo.project.name === OPENFIN_PROJECT_NAME) {
+      const mainWindow = creditPagesRec["mainWindow"]
 
-      if (testInfo.project.name === OPENFIN_PROJECT_NAME) {
-        const mainWindow = creditPagesRec["mainWindow"]
+      await mainWindow.evaluate(async () => {
+        window.fin.Window.getCurrentSync().maximize()
+      })
 
-        await mainWindow.evaluate(async () => {
-          window.fin.Window.getCurrentSync().maximize()
-        })
+      newRfqPage = creditPagesRec["credit-new-rfq"]
+      rfqsPage = creditPagesRec["credit-rfqs"]
+      rfqBlotterPage = creditPagesRec["credit-blotter"]
+    } else {
+      const pages = context.pages()
 
-        newRfqPage = creditPagesRec["credit-new-rfq"]
-        rfqsPage = creditPagesRec["credit-rfqs"]
-        rfqBlotterPage = creditPagesRec["credit-blotter"]
-      } else {
-        const pages = context.pages()
+      newRfqPage = pages.length > 0 ? pages[0] : await context.newPage()
 
-        newRfqPage = pages.length > 0 ? pages[0] : await context.newPage()
+      await newRfqPage.goto(`${process.env.URL_PATH}/credit`)
 
-        await newRfqPage.goto(`${process.env.URL_PATH}/credit`)
+      rfqsPage = newRfqPage
+      rfqBlotterPage = newRfqPage
+    }
+  })
 
-        rfqsPage = newRfqPage
-        rfqBlotterPage = newRfqPage
-      }
+  test.describe("New RFQ", () => {
+    test("When I select Googl instrument and click Send RFQ button then I should see a GOOGL RFQ created on the RFQ sections and I can accept any value @smoke", async () => {
```

Comment:
test label is way too long, isn't it?
"Create RFQ for GOOGL" should be enough ..